module/programs/helper.js

```diff
@@ -141,7 +156,39 @@ module.exports = class ProgramsHelper {
           isAPrivateProgram: data.isAPrivateProgram ? data.isAPrivateProgram : false,
         };
 
-        let program = await database.models.programs.create(programData);
+        if (checkDate) {
```

Comment:
@praveenKDass please add comments to functions to increase the readability

Reply:
Adding comments to increase readability @VISHNUDAS-tunerlabs